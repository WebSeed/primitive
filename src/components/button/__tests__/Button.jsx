import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import { Button } from '..'

describe('Button', () => {
  const setup = testProps => {
    const props = {
      ...testProps,
    }
    return shallow(<Button {...props}>Test button</Button>)
  }

  it('renders matching snapshot', () => {
    const $ = setup()

    expect(toJson($)).toMatchSnapshot()
  })
})
