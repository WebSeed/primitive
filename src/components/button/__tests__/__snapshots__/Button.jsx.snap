// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders matching snapshot 1`] = `
<button
  type="button"
>
  Hello
</button>
`;
